src/static/js/domline.js

```diff
@@ -61,6 +61,9 @@ domline.createDomLine = (nonEmpty, doesWrap, optBrowser, optDocument) => {
 
   if (document) {
     result.node = document.createElement('div');
+    // Set the ARIA role and label for JAWS compatibility. Do it here, because it is only needed and
```

Comment:
This comment says "Set the ARIA role" but I don't see `role` set anywhere. Is that intentional?

Reply:
The comment points to the attribute 'aria-live'='assertive', you are right, it has nothing to do with the role. So the comment is not well-formed and may be changed to better understanding.